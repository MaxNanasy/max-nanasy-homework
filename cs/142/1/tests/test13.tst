void myProc(){	get(a[0], vara[2]);}main(){}