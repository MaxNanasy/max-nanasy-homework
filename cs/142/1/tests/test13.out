 VOID  ident  (  )  {  GET  (  ident  [  number  ]  ,  ident  [  number  ]  )  ;  }  MAIN  (  )  {  }  <EOT> 